tools/tplg_parser/CMakeLists.txt

```diff
@@ -35,14 +35,16 @@ target_include_directories(sof_tplg_parser PRIVATE ${CMAKE_CURRENT_SOURCE_DIR}/i
 target_include_directories(sof_tplg_parser PRIVATE ${sof_source_directory}/src/include)
 target_include_directories(sof_tplg_parser PRIVATE ${sof_source_directory}/src/arch/host/include)
 target_include_directories(sof_tplg_parser PRIVATE ${sof_source_directory}/src/platform/library/include)
+target_include_directories(sof_tplg_parser PRIVATE /usr/include/alsa/sound/uapi)
+
 
 # TODO: The topology parser should NOT need to include RTOS header. FIX.
 target_include_directories(sof_tplg_parser PRIVATE ${sof_source_directory}/xtos/include)
 
 # TODO: add IPC4 option when it's ready.
 target_compile_options(sof_tplg_parser PRIVATE
 	-g -O -Wall -Werror -Wl,-EL -fPIC -DPIC
-	-Wmissing-prototypes -Wimplicit-fallthrough
+	-Wmissing-prototypes
```

Comment:
We're moving away from ancient `xt-xcc` versions for current products. How about never using ancient toolchains for the testbench? They should not be required for a test suite. This warning is important.

Reply:
I need a way to test HiFi3 code, we have lots of it.  We would need to convert the core configuration files for TGL from RG-2017.8 to current tools like RI-2022.10 and I doubt such task is planned. What is your suggestion to solve this?

Can cmake somehow disable this option for 2017.8-linux?

Reply:
Thanks, I'll try it. I really appreciate your helpful comments!